pygmt/helpers/utils.py

```diff
@@ -711,3 +711,120 @@ def args_in_kwargs(args: Sequence[str], kwargs: dict[str, Any]) -> bool:
     return any(
         kwargs.get(arg) is not None and kwargs.get(arg) is not False for arg in args
     )
+
+
+def sequence_join(
```

Comment:
Keep this as a private function?

```suggestion
def _sequence_join(
```

Reply:
This function is used in `grdclip.py` and will likely be used in many other functions. So, I'd prefer to make it public.